### PR TITLE
BZ 1970927: Release Note for Cronjobs will be GA from OCP 4.8

### PR DESCRIPTION
**Pull request by mburke5678**
Release note for https://github.com/openshift/openshift-docs/pull/33441

Preview: https://deploy-preview-33697--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-8-nodes-cronjob-qa_release-notes

### CODE REVIEW

**Comment by mburke5678** [comment]
@sunilcio Sorry. I should have asked you to review the release note for https://github.com/openshift/openshift-docs/pull/33441 the other day. Do you mind taking a quick look?

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -560,10 +560,14 @@ You can now specify an individual repository within a registry when creating lis
 
 For more information, see xref:../openshift_images/image-configuration.adoc#images-configuration-allowed_image-configuration[Adding specific registries] and xref:../openshift_images/image-configuration.adoc#images-configuration-blocked_image-configuration[Blocking specific registries].
 
+[id="ocp-4-8-nodes-cronjob-qa_{context}"]
+==== Cron jobs are generally available
+
+The cron job resource is now generally available. As part of this change, a new controller has been inplemented that substantially improves the performance cron jobs. For more information on cron jobs, see xref:../nodes/jobs/nodes-nodes-jobs.adoc#nodes-nodes-jobs-about_nodes-nodes-jobs[Understanding jobs and cron jobs].
```
Comment (sunilcio) [inline release_notes/ocp-4-8-release-notes.adoc:566]: Small spelling mistake for implemented :)

**Comment by sunilcio** [comment]
LGTM

**Comment by bmcelvee** [comment]
LGTM!

**Pull request merged**